static/io/include/mapbox/io.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
Why this directory static?

Reply:
This was due to the fact that `io` depends on some header-only libraries from `mapbox-base-extras`. I was with the impression that these were not "seen" by `mapbox` folder, but I was mistaken - `extras` subdirectory is actually added _before_ `mapbox`, so we're good - I'll move these back into `mapbox` then :+1: